src/Arcus.Messaging.Tests.Integration/ServiceBus/ServiceBusClient.cs

```diff
@@ -33,62 +35,108 @@ public ServiceBusClient(KeyRotationConfig configuration, ITestOutputHelper logge
             _logger = logger;
         }
 
+        /// <summary>
+        /// 
```

Comment:
Can you delete or provide these please?

Reply:
Ah, yes, was a reminder that I don't forget. Thanks!